src/components/ProductCategory/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Heading, UnorderedList } from '@chakra-ui/react';
+import { ProductItem } from '@/components/ProductItem';
+
+type CategoryWithItems = {
+    title: string,
+    items: string[],
+}
+
+export const ProductCategory = (productCategory: CategoryWithItems) => {
+    const { title, items } = productCategory;
+    return (<Box m='47px 89px 47px 80px'>
```

Comment:
Again, this indentation is wrong, the right way is like below example

```suggestion
    return (
      <Box m='47px 89px 47px 80px'>
```